### ISSUE TITLE
Could a 64-bit-executable version of the OEM installer be available

### ISSUE THREAD

**Issue by aklassen-appneta** [nmap/npcap#768]
**Describe the bug**
The current oem npacp installer is a 32-bit executable. Some of our customers object to any 32-bit-running things on their Windows machines.

**To Reproduce**
Steps to reproduce the behavior: With a policy in-place not to run 32-bit executables, try to launch the oem installer. It will not launch because it is not a 64-bit executable.

**Expected behavior**
A user with a policy against running 64-bit, but who needs a 64-bit component hidden in a 32-bit executable install set wants the package to install its 64-bit-executable contents.

**Diagnostic information**
* we had to update our NSIS-based installer to be a 64-bit executable to satisfy a customer with such a policy
* it required nothing more than adding the line
Target amd64-unicode
  to our NSI script.

**Additional context**
I understand that the installer may also contain 32-bit binaries but I haven't access to any 32-bit machines anymore so as to prove it. So I understand that the OEM installer MUST (for the sake of backward compatibility) be available as a 32-bit executable. What I am asking is that an OEM installer as 64-bit NSI executable be provided -- or that you would point out where the sources are for such an installer so we can patch it ourselves. I have been looking for such sources and have been so far unable to locate them.

**Comment by fyodor** [comment]
Hi @aklassen-appneta.  Thanks for the suggestion.  The Npcap installer itself is 32-bit for maximum compatibility while it installs native drivers and DLL's for the target architectures (x86, x64, ARM64).  Did your customer mention how they set these policies and why?  I'm not aware of any Windows features to remove x86 compatibility, though you could probably work up some custom Applocker or Group Policy or Windows Defenders rules to achieve this.  Understanding the mechanism they used and their reasoning for doing so would help in finding an appropriate resolution or workaround.

**Comment by aklassen-appneta** [comment]
No, they did not. It might simply have been a human-administered policy within the IT department. But it was presented to us in the past as a hard-stop by an important customer, so I'm looking for options.

**Comment by guyharris** [comment]
Could a Windows Installer package be provided? Or is there stuff that the Npcap installer needs to do that can't be handled by Windows Installer?

**Comment by fyodor** [comment]
@guyharris - that would be a good solution.  We are considering moving to (or at least offering) an MSI package version ([Issue #103](https://github.com/nmap/npcap/issues/103)).

**Comment by aklassen-appneta** [comment]
offering a 64-bit oem installer AS WELL AS the current one would be easier than moving toward an MSI, but I can see why you would want to go there. My own preference would be the existing NSIS based installer with the one tweak, though. We use an NSIS based one for our own product and I have yet to see a compelling technical case for leaving that behind. I'll examine #103 to see if one is provided there.
